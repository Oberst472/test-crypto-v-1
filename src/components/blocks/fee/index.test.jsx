import {render, screen} from '@testing-library/react'

import BlockFee from './index'

describe('BLOCK FEE TEST', () => {
    test('BlockFee renders without props',
        () => {
            render(<BlockFee
                className={null}
                c14Fee={null}
                networkFee={null}
                totalFee={null}
            >
                Fees
            </BlockFee>)
            const el = screen.getByTestId('block-fee')
            expect(el).toBeInTheDocument()
        })
    test('BlockFee snapshot',
        () => {
            // eslint-disable-next-line testing-library/render-result-naming-convention
            const component = render(<BlockFee
                className={null}
                c14Fee={null}
                networkFee={null}
                totalFee={null}
            >
                Fees
            </BlockFee>)
            expect(component).toMatchSnapshot()
        })
})
