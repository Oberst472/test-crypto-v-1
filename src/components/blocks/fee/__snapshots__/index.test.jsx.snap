// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BLOCK FEE TEST BlockFee snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="block-fee"
        data-testid="block-fee"
      >
        <span
          class="block-fee__title"
        >
          Fees
        </span>
        <span
          class="block-fee__box"
        >
          <span
            class="block-fee__subtitle"
          >
            Network Fee
          </span>
          <span
            class="block-fee__val"
          >
            0
          </span>
        </span>
        <span
          class="block-fee__symbol"
        >
          +
        </span>
        <span
          class="block-fee__box"
        >
          <span
            class="block-fee__subtitle"
          >
            c14 Fee
          </span>
          <span
            class="block-fee__val"
          >
            0
          </span>
        </span>
        <span
          class="block-fee__symbol"
        >
          =
        </span>
        <span
          class="block-fee__box"
        >
          <span
            class="block-fee__subtitle"
          >
            Total Fee
          </span>
          <span
            class="block-fee__val"
          >
            0
          </span>
        </span>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="block-fee"
      data-testid="block-fee"
    >
      <span
        class="block-fee__title"
      >
        Fees
      </span>
      <span
        class="block-fee__box"
      >
        <span
          class="block-fee__subtitle"
        >
          Network Fee
        </span>
        <span
          class="block-fee__val"
        >
          0
        </span>
      </span>
      <span
        class="block-fee__symbol"
      >
        +
      </span>
      <span
        class="block-fee__box"
      >
        <span
          class="block-fee__subtitle"
        >
          c14 Fee
        </span>
        <span
          class="block-fee__val"
        >
          0
        </span>
      </span>
      <span
        class="block-fee__symbol"
      >
        =
      </span>
      <span
        class="block-fee__box"
      >
        <span
          class="block-fee__subtitle"
        >
          Total Fee
        </span>
        <span
          class="block-fee__val"
        >
          0
        </span>
      </span>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
